workshop1/03-connecting-the-api.md

```diff
@@ -135,6 +136,12 @@ as application state, and is fetched when the page first loads.
 3. Update the template returned by your app component to reference the `pets`
    state variable instead of `mockedPets`.
 
+4. Navigate to the node terminal, within Visual Studio Code. You will notice a yellow message saying your code was `Compiled with warnings`. Because we've changes a few things during this step, there are some variables we are not using anymore: `logo` and `mockedPets`.  
+Even though they don't have any effect on the code, it's good practice to keep your code as clean as possible, and it includes making sure you don't have any unused code (which will clutter your project as it accumulates, and don't bring any value).  
+_Always keep a look at this section of your project: ideally we want our projects to be warning-free (even though sometimes that's not possible)._
```

Comment:
Small language tweak: I'd probably say "keep an eye on" rather than "keep a look at". Or perhaps "Remember to look at ... occasionally."

Reply:
Changed it to "keep an eye on" (also updated my internal phrasal verbs repository :P )
Thanks!